crates/sui-types/src/deny_list_v2.rs

```diff
@@ -20,6 +20,33 @@ use serde::{Deserialize, Serialize};
 use std::collections::{BTreeMap, BTreeSet};
 use std::fmt;
 
+pub(super) mod constants {
+    const ADDRESS: usize = 32;
+    const ID: usize = ADDRESS;
+    const UID: usize = ID;
+    const STR: usize = 128;
+    const TYPE_NAME: usize = ADDRESS + STR + STR + 4;
+    const BOOL: usize = 1;
+    const U64: usize = 8;
+    const OPTION_SOME_BOOL: usize = BOOL + 1;
+    const OBJECT_METADATA: usize = 9 + TYPE_NAME;
+
+    const CONFIG_KEY: usize = U64 + TYPE_NAME;
+    const ADDRESS_KEY: usize = ADDRESS;
+
+    // config dynamic object field
+    pub const CONFIG_DOF_FIELD: usize = OBJECT_METADATA + UID + CONFIG_KEY + ID;
+
+    // setting dynamic field
+    const SETTING_DATA: usize = U64 + OPTION_SOME_BOOL + OPTION_SOME_BOOL;
+    const OPTION_SOME_SETTING_DATA: usize = SETTING_DATA + 1;
+    const SETTING: usize = OPTION_SOME_SETTING_DATA;
+    pub const SETTING_FIELD: usize = OBJECT_METADATA + UID + ADDRESS_KEY + SETTING;
+}
+
+pub const CONFIG_DYNAMIC_OBJECT_FIELD_SIZE: usize = constants::CONFIG_DOF_FIELD;
+pub const CONFIG_SETTING_DYNAMIC_FIELD_SIZE: usize = constants::SETTING_FIELD;
```

Comment:
@lxfind happy to fix these as you see fit. 
Right now they are about 30% larger than the size of the dynamic fields in our tests, as am assuming the worst case size for the identifiers.